integration_tests/src/main/python/generate_expr_test.py

```diff
@@ -214,3 +214,19 @@ def test_generate_outer_fallback():
         lambda spark: spark.sql("SELECT array(struct(1, 'a'), struct(2, 'b')) as x")\
             .repartition(1).selectExpr("inline_outer(x)"),
         "GenerateExec")
+
+# stack not guarantee to produce the same output order as Spark does
+@ignore_order(local=True) 
+def test_stack():
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark : spark.range(100).selectExpr('*', 'stack(3, id, 2L, 3L, 4L, 5L, 6L)'))
+
+# stack not guarantee to produce the same output order as Spark does
+@ignore_order(local=True)
+def test_stack_mixed_types():
```

Comment:
need tests for mixed expressions. (Stack and other generators), because this PR is going to run other generators with `GpuExpandExec`.